specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/examples/questionanswering/SuccessfulQueryKnowledgebases.json

```diff
@@ -6,7 +6,7 @@
     "api-version": "2021-05-01-preview",
```

Comment:
All these samples' api-versions need to be updated.

Reply:
Thanks for pointing it out, will do in the next PR.